h2o-genmodel/src/main/java/hex/genmodel/tools/PrintMojo.java

```diff
@@ -68,6 +71,10 @@ private static void usage() {
     System.out.println("");
     System.out.println("    --fontsize | -f    Set font sizes of strings.");
     System.out.println("");
+    System.out.println("    --raw    Print raw graph representation to standard output.");
+    System.out.println("");
+    System.out.println("    --internal    Internal H2O representation of the graph (splits etc.) is used for generating the GRAPHVIZ format.");
```

Comment:
What does adding `internal` mean to for the output?

Reply:
Internal H2O representation of the graph. Currently, internal representations of the splits. It will print out numerical representation of categorical splits too, together with categorical level labels on the edges.

You can see an example of the behavior in this PR's description (it is up to date).

Reply:
This is the functionality you requested. I'll push a better documentation.